<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>netDlg</class>
 <widget class="QDialog" name="netDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>781</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>IP地址增加</string>
   </property>
   <widget class="QLabel" name="ip_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>35</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maskip_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>95</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>子网掩码:</string>
    </property>
   </widget>
   <widget class="QLabel" name="gateip_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>155</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>默认网关:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>增加</string>
    </property>
    <property name="icon">
     <iconset resource="fastvideo.qrc">
      <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
    </property>
   </widget>
   <widget class="MyIPAddressEdit" name="IPEdit" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>42</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="MyIPAddressEdit" name="MaskIPEdit" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>96</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="MyIPAddressEdit" name="GateIPEdit" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>155</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>781</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>当前IP地址</string>
   </property>
   <widget class="QLabel" name="ip_label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maskip_label_2">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>子网掩码:</string>
    </property>
   </widget>
   <widget class="QLabel" name="gateip_label_2">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>30</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>默认网关:</string>
    </property>
   </widget>
   <widget class="QLabel" name="curip_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>192.168.1.98</string>
    </property>
   </widget>
   <widget class="QLabel" name="curmaskip_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>255.0.0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="curgateip_label">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>192.168.1.1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>781</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>IP地址设置</string>
   </property>
   <widget class="QTableWidget" name="ip_tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>39</y>
      <width>571</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ip地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>子网掩码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>默认网关</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>115</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>移除</string>
    </property>
    <property name="icon">
     <iconset resource="fastvideo.qrc">
      <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="configBtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>155</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
    <property name="icon">
     <iconset resource="fastvideo.qrc">
      <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>智能设置</string>
   </property>
   <property name="icon">
    <iconset resource="fastvideo.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyIPAddressEdit</class>
   <extends>QWidget</extends>
   <header>myipaddressedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
